src/main/java/org/recordrobotics/charger/control/DoubleControl.java

```diff
@@ -21,12 +21,12 @@ public double getDriveLong() {
 
 	@Override
 	public double getDriveLat() {
-		return -_gamepad2.getLeftX();
+		_gamepad2.getStartButton(); // line here to avoid unused variable error
```

Comment:
It's generally better to PMD suppress something like this rather than write arbitrary code

Reply:
it works for now

Reply:
Just not good practice

Reply:
I shall fix it